marconi-sidechain-experimental/src/Marconi/Sidechain/Experimental/Api/Types.hs

```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Marconi.Sidechain.Experimental.Api.Types (
+  Types.configPort,
+  Types.configTrace,
+  module Marconi.Sidechain.Experimental.Api.Types,
+) where
+
+import Cardano.Api qualified as C
+import Control.Lens (makeLenses, (^.))
+import Control.Monad.Except (ExceptT, mapExceptT)
+import Control.Monad.Reader (ReaderT, withReaderT)
+import Data.List.NonEmpty (NonEmpty)
+import Marconi.ChainIndex.Api.Types qualified as Types
+
+{- SERVER CONFIG -}
+
+{- | 'SidechainHttpServerConfig' augments @Types.'HttpServerConfig'@ with
+ - 'SidechainExtraHttpServerConfig', containing anything needed for sidechain
+ - queries not included in HttpServerConfig.
+-}
+data SidechainHttpServerConfig = SidechainHttpServerConfig
+  { _chainIndexHttpServerConfig :: !Types.HttpServerConfig
+  , _sidechainExtraHttpServerConfig :: !SidechainExtraHttpServerConfig
```

Comment:
For now, since the "Extra" config has only one field, I agree with you that simply adding the field to `SidechainHttpServerConfig` is a cleaner approach. We can see if it continues to makes sense as we build out the experimental package.